spec/integration/app_host_source_dir/after/gen/Foo/Foo.xcodeproj

```diff
@@ -165,7 +165,9 @@ Targets:
           CODE_SIGN_IDENTITY: iPhone Developer
           INFOPLIST_FILE: "${SRCROOT}/App-iOS/Info.plist"
           IPHONEOS_DEPLOYMENT_TARGET: '9.0'
-          LD_RUNPATH_SEARCH_PATHS: "$(inherited) @executable_path/Frameworks"
+          LD_RUNPATH_SEARCH_PATHS:
```

Comment:
I believe those changed because the "after" Xcodeproj object version changed its default now to 50 (9.3). Xcode probably treats those as arrays now.